osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -120,8 +120,14 @@ protected override void LoadComplete()
 
             State.ValueChanged += state =>
             {
+                // apply any custom values
+                ApplyCustomValues?.Invoke(this);

```

Comment:
this shouldn't be necessary (can use the already existing `ApplyToDrawableHitObjects` instead)

Reply:
Can you elaborate what you thought? I don't understand.